bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java

```diff
@@ -1060,4 +1061,36 @@ public boolean isEnsembleAdheringToPlacementPolicy(List<BookieSocketAddress> ens
         }
         return true;
     }
+
+    @Override
+    public boolean areAckedBookiesAdheringToPlacementPolicy(Set<BookieSocketAddress> ackedBookies,
+                                                            int writeQuorumSize,
+                                                            int ackQuorumSize) {
+        HashSet<String> rackCounter = new HashSet<>();
+        int minWriteQuorumNumRacksPerWriteQuorum = Math.min(writeQuorumSize, minNumRacksPerWriteQuorum);
+
+        ReentrantReadWriteLock.ReadLock readLock = rwLock.readLock();
+        readLock.lock();
+        try {
+            for (BookieSocketAddress bookie : ackedBookies) {
+                try {
+                    rackCounter.add(knownBookies.get(bookie).getNetworkLocation());
+                } catch (Exception e) {
```

Comment:
Nit: why catch a generic Exception?

Reply:
Fixed. Removed the try catch as it not needed.